runtime/bifrost-polkadot/src/xcm_config.rs

```diff
@@ -503,14 +503,37 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 	}
 }
 
+/// Matches foreign assets from a given origin.
+/// Foreign assets are assets bridged from other consensus systems. i.e parents > 1.
+pub struct IsForeignNativeAssetFrom<Origin>(PhantomData<Origin>);
+impl<Origin> ContainsPair<MultiAsset, MultiLocation> for IsForeignNativeAssetFrom<Origin>
```

Comment:
Not all foreign assets are bridged from other ecosystems.

I suggest you change this comment and structure name to something like:

```suggestion
/// Matches bridged assets from a given origin.
/// Assets bridged from other consensus systems. i.e parents == 2.
pub struct IsBridgedAssetFrom<Origin>(PhantomData<Origin>);
impl<Origin> ContainsPair<MultiAsset, MultiLocation> for IsBridgedAssetFrom<Origin>
```